tests/test_lib.py

```diff
@@ -510,5 +526,6 @@ def test_license_parsing_unknown(make_dist):
 def test_metadata_cant_be_extracted(make_dist, mocker):
     make_dist()
     mocker.patch("pkginfo.get_metadata", return_value=None)
-    with pytest.raises(Exception("failed to get metadata")):
+    with pytest.raises(Exception) as ex:
         JohnnyDist("jdtest")
+    assert str(ex.value) == "failed to get metadata"
```

Comment:
Why modifying this test?  You may have forgot to install a [new development dependency](https://github.com/wimglenn/johnnydep/blob/71b51d389ef28cc22b84a5c3f46e4f635b4d99c0/requirements-dev.txt#L5) that was added in #76